graylog2-server/src/main/java/org/graylog2/streams/StreamRouterEngine.java

```diff
@@ -329,6 +333,9 @@ public Stream call() throws Exception {
             } catch (Exception e) {
                 LOG.warn("Unexpected error during stream matching", e);
```

Comment:
since the original issue was about improving the log message,
we should log the improved `error` text here as well